codegen/cmd/injection-gen/generators/reconciler_controller_stub.go

```diff
@@ -106,15 +118,27 @@ func NewController(
 
 	{{.type|lowercaseSingular}}Informer := {{.informerGet|raw}}(ctx)
 
+	{{if .hasClass}}
+	classValue := "default" // TODO: update this to the appropriate value.
+	classFilter := {{.annotationFilterFunc|raw}}({{.classAnnotationKey|raw}}, classValue, false /*allowUnset*/)
+	{{end}}
+
 	// TODO: setup additional informers here.
-	{{if .hasClass}}// TODO: pass in the expected value for the class annotation filter.{{end}}
+	{{if .hasClass}}// TODO: remember to use the classFilter from above to filter appropriately.{{end}}
 
 	r := &Reconciler{}
-	impl := {{.reconcilerNewImpl|raw}}(ctx, r{{if .hasClass}}, "default"{{end}})
+	impl := {{.reconcilerNewImpl|raw}}(ctx, r{{if .hasClass}}, classValue{{end}})
 
 	logger.Info("Setting up event handlers.")
 
+	{{if .hasClass}}
+	{{.type|lowercaseSingular}}Informer.Informer().AddEventHandler({{.filterHandler|raw}}{
+		FilterFunc: classFilter,
+		Handler:    controller.HandleAll(impl.Enqueue),
```

Comment:
I like this, we should definitely do this regardless.  thanks!

Reply:
yea nice.